src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.ui.Ui;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+/**
+ * It is a super class that named Command for the Duke program.
+ * When the parser calls it, it will receive the requests from the users
+ * during the running of the program and starts to call its subclasses for
+ * execution.
+ */
+public class Command {
+    protected String userMessage;
+    protected static boolean exit;
```

Comment:
Maybe you could name this boolean as isExit to make it more obvious that it's a boolean variable.

Reply:
I've changed it in the latest commit. Thx.